inst/private/check_and_convert.m

```diff
@@ -0,0 +1,38 @@
+function obj = check_and_convert()
+  newl = sprintf('\n');
+  
+  str_check_is_list = strjoin({'if isinstance(_outs, (list, tuple)):',
+                              '  is_list = True',
+                              'else:',
+                              '  is_list = False'},newl);
+  pyexec(str_check_is_list);
+  is_list = pyeval('is_list');
+  
+  if is_list
+    n = pyeval('len(_outs)');
+  else
+    n = 1;
+  end
+  
+  obj = [];
```

Comment:
> > +  obj = [];
>
> Does this work for you? shouldn't obj be a cell array? It needs to be
heterogeneous...
>

Sorry. 'cell array'?
And yes it worked for me. But I tested only the conditions where either one
element was returned or a list with elements of same type were returned.